src/System.Private.ServiceModel/tools/IISHostedWcfService/Web.config

```diff
@@ -0,0 +1,95 @@
+<?xml version="1.0" encoding="utf-8"?>
+ <!--Copyright (c) Microsoft Corporation.  All Rights Reserved.-->
+<configuration>
+  <system.diagnostics>
+    <sources>
+      <source name="System.ServiceModel" switchValue="Information, ActivityTracing" propagateActivity="true">
+        <listeners>
+          <add name="traceListener" type="System.Diagnostics.XmlWriterTraceListener" initializeData="c:\log\Traces.svclog" />
```

Comment:
I believe the same trace files can be shared by multiple service instances, i.e. they will co-exist and can be filtered out by process IDs. 

Shall I change the default value from Information to Warning to avoid the trace file growing too fast?